pkgs/applications/terminal-emulators/foot/default.nix

```diff
@@ -165,10 +165,10 @@ stdenv.mkDerivation {
     meson configure -Db_pgo=generate
     ninja
     # make sure there is _some_ profiling data on all binaries
+    ninja test
```

Comment:
`ninja test` is just a wrapper around `meson test`, so it probably makes slightly more sense to use the latter (which has various options that can't be passed through ninja).  It's what we do in the setup hooks.

Reply:
done.